addons/mail/models/mail_thread.py

```diff
@@ -2466,6 +2467,7 @@ def message_notify(self, *,
         if not subtype_id:
             subtype_id = self.env['ir.model.data']._xmlid_to_res_id('mail.mt_note')
 
+        body = Markup() + body
```

Comment:
Why is it different from message_post ? They are globally the same kind of methods.L

Reply:
no reason, it's because I have finally learned how to properly use escape when I reached that part of the file